Finance/transferwise-currency-tracker.1m.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# <bitbar.title>Currency Tracker Transferwise</bitbar.title>
+# <bitbar.version>1.0</bitbar.version>
+# <bitbar.author>Andrew Keating</bitbar.author>
+# <bitbar.author.github>andrewzk</bitbar.author.github>
+# <bitbar.desc>Keep an eye on Transferwise currency exchange rates</bitbar.desc>
+# <bitbar.dependencies>python</bitbar.dependencies>
```

Comment:
It'd be great if you could add an image under the `bitbar.image` metatag. :smile:. Have a look at the [bitbar metadata for reference](https://github.com/matryer/bitbar#metadata)